src/features/movies/MovieList/MovieSearchPage/index.js

```diff
@@ -0,0 +1,63 @@
+import { useSelector } from "react-redux";
+import { Link } from "react-router-dom";
+import { selectData, selectInputValue } from "../../../../Navigation/Search/searchSlice";
+import { Section, SectionTitle } from "../../../../common/Section/Section";
+import { posterURL } from "../../../../utils/API/APIURLS";
+import { Wrapper } from "./styled";
+import { NoMovieIcon, SectionWrapper, StyledLink, Tile, TileContent, TileTitle, TileSubTitle, TileTags, TileTag, TileImageContainer, TileImage, Rating, } from "../../MovieList/styled";
+import errorImage from "./noResult.png";
```

Comment:
We have that grafic as svg in assets folder, I think we should use SVGs where possible 😉 You can import that one from src/assets/icon-empty.svg

Reply:
thanks i changed it ;)